cincinnati/src/plugins/internal/node_remove.rs

```diff
@@ -85,19 +85,23 @@ mod tests {
             let metadata: TestMetadata = vec![
                 (
                     0,
-                    hashmap! { format!("{}.{}", key_prefix, key_suffix) => String::from("true") },
+                    hashmap! { format!("{}.{}", key_prefix, key_suffix) => String::from("true") }
```

Comment:
Shouldn't we use `btreemap!` here?

Reply:
I honestly didn't look for its existence. `anymap!` would be best :-D

Reply:
I decided to remove the macro invocation and the maplit dependency altogether.